ci/environment-dev-3.6.yml

```diff
@@ -36,3 +36,4 @@ dependencies:
   - pip:
     - sphinx_copybutton
     - sat-stac
+    - git+https://github.com/martindurant/intake.git@explicit_driver
```

Comment:
I am happy to merge this, if it works for you. Will tests passing in this PR be sufficient proof, do you think?

Reply:
yes, that was my plan. tests are passing locally so I think we should be good.